Cargo.lock

```diff
@@ -65,6 +65,12 @@ dependencies = [
  "rand_xorshift",
 ]
 
+[[package]]
+name = "allocator-api2"
+version = "0.2.14"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "c4f263788a35611fba42eb41ff811c5d0360c58b97402570312a350736e2542e"
+
```

Comment:
Hmm no, it's not. Let me double-check.

Reply:
So here's what happening:
- hashbrown uses the [new Cargo feature syntax](https://rustwiki.org/en/cargo/reference/features.html#dependency-features) (since 1.60) to enable the `nightly` feature in `allocator-api2` when the `nightly` feature in hashbrown is enabled, but without forcing the `allocator-api2` optional dependency to be enabled. The relevant line in hashbrown's Cargo.toml is here: https://github.com/rust-lang/hashbrown/blob/3056ee976adaa39f5b502db7441b33ffab372ab6/Cargo.toml#L45
- Due to https://github.com/rust-lang/cargo/issues/10801, `allocator-api2` is included in Cargo.lock even though it is not built. This seems to be a limitation of Cargo's resolver, though it may be fixable in the future.

My suggestion would be to just whitelist the `allocator-api2` dependency for now, with a note that it's just there due to a Cargo bug, and is not actually built.